CHANGES.md

```diff
@@ -1,5 +1,65 @@
 # Changelog
 
+## 6.1 (2023-11-03)
+[View detailed changelog](https://github.com/spinalcordtoolbox/spinalcordtoolbox/compare/6.0...6.1)
+
+**FEATURE**
+ - **sct_analyze_lesion**: Add function to output the axial damage ratio + minor improvements. [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4207)
+ - **sct_download_data**: Color installed datasets for `sct_download_data -h`. [View pull request](https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4257)
```

Comment:
Looking back, this should probably not be in the "FEATURE" category. (Its sister PR is under the "Documentation" section.)

Reply:
Done!